Rakefile

```diff
@@ -28,7 +28,7 @@ end
 
 def create_simple_logger
   logfile = Tempfile.new('my_test_log.log')
-  ::Logger.new(logfile.path)
+  Logger.new(logfile.path)
```

Comment:
I think this is here for something like preventing standard library logger to get picked instead of such thing, any idea if this change is cool?

Reply:
(btw, here is a rake file to run benchmarks so I don't think it matters)

Reply:
oh, ok, yeah did not notice the file, if this makes Rubocop happier then keep the change.

Reply:
dropped the commit